paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/SparkSource.scala

```diff
@@ -82,24 +82,6 @@ class SparkSource
     SparkSource.toBaseRelation(table, sqlContext)
   }
 
-  private def loadTable(options: JMap[String, String]): DataTable = {
-    val path = CoreOptions.path(options)
-    val catalogContext = CatalogContext.create(
-      Options.fromMap(
-        mergeSQLConfWithIdentifier(
-          options,
-          extractCatalogName().getOrElse(NAME),
```

Comment:
The `extractCatalogName()` and `NAME` here actually has no meaning and has nothing to do with the catalog name